docs/guides/first-experiment.mdx

```diff
@@ -47,11 +47,11 @@ You should see that the experiment group indeed has a 25% rollout, with some spe
 
 ![Screen Shot 2021-06-15 at 11 42 09 AM](https://user-images.githubusercontent.com/82126616/122106466-ce1e9580-cdce-11eb-82d3-aa270602bbd6.png)
 
-### Step 4: Check Metrics Impact! {#checkmetrics}
```

Comment:
keep the {#checkmetrics} - we add those tags to deeplink directly to this header.  It shouldnt show on the page anywhere, but might break a link if we remove it